stories/Components/UIcomponents/Accordion/Accordion.jsx

```diff
@@ -9,7 +9,7 @@ export const Accordion = ({ headerText, descriptionText }) => {
   // }, []);
 
   return (
-    <ul className="accordion" aria-label="accordion" data-component="accordion">
+    <ul className="accordion" aria-label="accordion" data-undps-component="accordion">
```

Comment:
@mihir880 , it has to be `data-undpds-` prefix (as in **UNDP** **D**esign **S**ystem) so it will be mnemonic